src/main/java/graphics/Controller.java

```diff
@@ -159,6 +162,60 @@ public void saveGraphHandler(ActionEvent actionEvent) {
   }
 
   /**
+   * Displays an alert listing all supported command line commands.
+   */
+  @SuppressWarnings("checkstyle:LineLength")
+  public void displayHelpAlert() {
+    Alert alert = new Alert(Alert.AlertType.INFORMATION);
+    alert.setTitle("Supported Commands");
+    alert.setHeaderText(null);
+    alert.setResizable(true);
+    alert.getDialogPane().setMinWidth(750);
+
+    alert.setContentText(
```

Comment:
Could this be done so that the text is highlightable and the command names are in bold?